pkg/transformer/metrics_transformer.go

```diff
@@ -0,0 +1,215 @@
+package transformer
+
+import (
+	"fmt"
+	"github.com/flipkart-incubator/ottoscalr/pkg/metrics"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"encoding/json"
+
+	"github.com/hashicorp/go-retryablehttp"
+)
+
+type OutlierInterval struct {
+	StartTime time.Time
+	EndTime   time.Time
+}
+
+type ALLEvents struct {
+	HasMore bool            `json:"hasMore,omitempty"`
+	Content []EventMetadata `json:"content,omitempty"`
+}
+
+type EventMetadata struct {
```

Comment:
This usecase may be applicable to outside FK also. We need to make the schema a bit more generic. Fk constructs like EA, prebook need to be hidden from this schema.